content/docs/learning-guide/tutorials/rendering/vegetation-bending-tutorial.md

```diff
@@ -0,0 +1,834 @@
+
+This video shows the motion vectors when `OUT.m_motion` is scaled by `10000.0`.
+{{< video src="/images/atom-guide/vegetation-bending-tutorial/motionvectortree.mp4" autoplay="true" loop="true" muted="true" width="100%" info="Three trees in the Editor swaying with detail bending, with the motion vector visualizer on the bottom right indicating the direction of movement." >}}
+
+The pass tree visualizer tool is also helpful with debugging shaders and passes, so you can see the output of certain steps of different passes when you select them in the **PassTree**.
+
+## Download the AtomTutorial Gem sample
+Now that you've completed this tutorial, you can compare your results to our working version of the vegetation bending material type in the **AtomTutorials Gem** in the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). You can either download and place the [final working vegetation bending files](https://github.com/o3de/sample-code-gems/tree/main/atom_gems/AtomTutorials/Assets/VegetationBending) in your project, or you can download the Gem and add it to the engine.
+
+If you'd like to download and enable the **AtomTutorials Gem**, do the following:
+1. Download or clone the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). 
+
+   {{< note >}}
+   If you followed this tutorial, you already downloaded or cloned this repository, but probably moved the `Object` files out of the repository. You can move the files back or re-download/clone the repository.
+   {{< /note >}}
+
+1. Open `VegetationBending.materialtype` and replace all the instances of `{your-path-to-o3de}` with your absolute path to O3DE. 
+
+1. [Register the **AtomTutorials Gem** to the engine](/docs/user-guide/project-config/register-gems). In the command line interface, `cd` into the engine. Then, run
+
+   ```
+   scripts\o3de register -gp {your-path-to-sample-code-gems}\atom_gems\AtomTutorials -espp {your-project-path}
+   ```
+
+   For example, `scripts\o3de register -gp C:\sample-code-gems\atom_gems\AtomTutorials -espp C:\MyProject`
+
+1. [Add the Gem into your project](/docs/user-guide/project-config/add-remove-gems). 
+
+   1. Open the **Project Manager**, which should be located in your project's `build/windows/bin/profile/o3de.exe`.
+
+   1. In the menu, find your project, click on the three bars, and select **Configure Gems...**.
+
+   1. In the **Gem Catalog**, search for **AtomTutorials** and enable the Gem.
+
+   1. Click **Save** in the bottom right.
```

Comment:
Since the linked page explains all of this, you can remove them here and consolidate into single step. 

```suggestion
1. Add the **AtomTutorials** Gem to you project. Follow the instructions in [Adding and Removing Gems in a Project](/docs/user-guide/project-config/add-remove-gems). 
```